docs/data/sdks/browser-2/migration.md

```diff
@@ -30,78 +28,19 @@ For Node projects, update your dependency list in package.json.
 
 ### Default events tracking
 
-Starting Browser SDK 2.0, default tracking is enabled by default. Default tracking is implicit tracking performed by Amplitude on your behalf, and includes page views, sessions, file downloads, form interactions, and marketing attribution.
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/cookie-options.mddefault-event-tracking.md"
 
-To opt out of default tracking, set `options.defaultTracking` to `false`.
+### Deprecates user agent client-side parsing
```

Comment:
i don't agree that this part should be this high on the page considering how little friction this has produced. other sections below are far more important

Reply:
I think this is the best place to add this content. 
I prefer to list all things instead of hiding them. Eventually, the customer will come back to us and we might not remember the changes and might take more time to investigate later. 
Any thoughts on this? @justin-fiedler @Mercy811  @ondrej

Reply:
@kevinpagtakhan I agree there hasn't been much customer friction on this but since it is a major change that affects all ingested events I think we should keep it near the top.

If we want to move it down, I think it should stay be above `### No to enums` and other "stylistic" changes.

Reply:
@yuhao900914 i wasn't trying to get this section removed 😅 just want to note that for any section we "prioritize", there are sections that get "deprioritized" in terms of page depth. i still believe the changes in implicit tracking, and configuration are more important. if not correctly migrated, implicit tracking could lead to increased costs for customers, incorrect configuration leading to data loss

@justin-fiedler Right before enums seems the perfect spot